packages/authentication/test/acceptance/basic-auth.ts

```diff
@@ -39,20 +45,19 @@ import {
 import {Strategy} from 'passport';
 import {BasicStrategy} from 'passport-http';
 
-const SequenceActions = CoreBindings.SequenceActions;
+const SequenceActions = RestBindings.SequenceActions;
```

Comment:
Should we have the generic `Sequence` and `Action` in the core so that they can be reused by both REST and gRPC? Each of them will have their own flavors.

Reply:
IIRC, when Miroslav and I chatted about it, the idea of what a `Sequence` is might not be the same between the two protocols, and it would be extremely difficult to make a highly-abstract approach that fits all needs without being overly verbose and difficult to work with.

Having their own Sequences means we're still laying out a typical pattern that we suggest for others, but doesn't necessarily lock component devs into this approach.

cc @bajtos

Reply:
Yes, each transport needs a very different `Sequence` contract. Right now, there is no overlap between gRPC and REST Sequence.

Reply:
The point of `Sequence` is that you implement your own opinionated version based on what the (in this new case) `Server` should do in response to a request.

We originally validated the `Sequence` design by using it to build out a webpack-dev server, something not restful at all. The flexibility is the point of `Sequence`.

For `RPC` you would certainly have a different `Sequence` than `REST`. Really, each app is likely to end up with their own `Sequence`. Its not meant to be an out of the box feature, rather we provide examples that make sense as starting points.